src/Compilers/Core/Portable/Symbols/Attributes/CommonEventWellKnownAttributeData.cs

```diff
@@ -1,16 +1,12 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
-using System.Diagnostics;
-using Microsoft.CodeAnalysis.Text;
-
 namespace Microsoft.CodeAnalysis
 {
     /// <summary>
     /// Information decoded from well-known custom attributes applied on an event.
     /// </summary>
-    internal class CommonEventWellKnownAttributeData : WellKnownAttributeData
+    internal class CommonEventWellKnownAttributeData : WellKnownAttributeData, ISkipLocalsInitAttributeTarget
```

Comment:
>ISkipLocalsInitAttributeTarget [](start = 79, length = 30)

where does the interface come into play? is it necessary?

Reply:
See `AttributeData.DecodeSkipLocalsInitAttribute<T>`. The interface is how we share functionality between the different AttributeData classes (like CommonEventWellKnownAttributeData, MethodWellKnownAttributeData, et al)